specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```
@@ -2232,11 +2232,11 @@
       "properties": {
         "timeAfterCreate": {
           "type": "string",
-          "description": "Time after creation to attempt rotate. It will be in ISO 8601 format. Example: 90 days : \"P90D\" "
+          "description": "Time after creation to attempt rotate. It only applies to rotate. It will be in ISO 8601 format. Example: 90 days : \"P90D\" "
```

This doesn't read right. Should be something like,
```suggestion
          "description": "Time after creation to attempt to rotate. It only applies to rotate. It will be in ISO 8601 duration format. Example: 90 days : \"P90D\" "
```

Also, ISO 8601 covers both time and duration so it's good to qualify (typically only "duration" is qualified that I've seen in RFCs and other docs).

---

Same changes in the other file as well.